modules/core-layers/src/path-layer/path-layer.js

```diff
@@ -144,7 +174,13 @@ export default class PathLayer extends Layer {
       dashJustified
     } = this.props;
 
-    this.state.model.render(
+    const {model} = this.state;
+
+    // Base layer sets instance count to `this.state.numInstances`
+    // We need to remove the last 3 to avoid accessing out of range vertices in offsetted attributes
+    model.setInstanceCount(this.state.numInstances - 3);
```

Comment:
The downside of offsetting attributes is that the length of the array no longer matches the number of instances to draw. The base layer uses `this.state.numInstances` for both attribute allocation and the draw method, and a manual override is necessary.